sql/2020/22_HTTP_2/percent_of_sites_affected_by_cdn_prioritization_issues.sql

```diff
@@ -0,0 +1,30 @@
+# standardSQL
+# % of sites affected by CDN prioritization issues (H2 and served by CDN)
+SELECT 
+  client,
+  IF(pages.cdn = "", "Not using CDN", pages.cdn) AS CDN,
+  IF(prioritization_status IS NOT null, prioritization_status, "Unknown") prioritizes_correctly,
+  COUNT(0) AS num_pages,
+  ROUND(COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY client), 4) AS pct
+FROM (
+    SELECT
+      date,
+      client,
+      url,
+      _cdn_provider as cdn
+    FROM 
+      `httparchive.almanac.requests` 
+    WHERE 
+      firstHtml AND
+      JSON_EXTRACT_SCALAR(payload, "$._protocol") ="HTTP/2" AND
+      date='2020-08-01'
+  ) AS pages
+LEFT JOIN 
+  `httparchive.almanac.h2_prioritization_cdns` AS h2_pri
+ON pages.cdn = h2_pri.cdn
+AND pages.date = h2_pri.date
```

Comment:
`USING` is helpful when the fields have the same name. This way you also don't need to alias the tables `h2_pri` and `pages`.
```suggestion
USING (cdn, date)
```

Reply:
Change made and syntax check passed